File: 2425/Python/primaEsercitazione.py
'''
Programma che valuta se un numero intero dato è 
maggiore o minore di 10.
Se è maggiore stampa i valori da 0 a 10
Se è minore stampa i valori da 10 a 0
(usando un cilco while o for)
'''

fix_number = 10
number = int(input("Inserisci un numero intero positivo: "))

if (number > fix_number):
    print("Stampo i valori da 0 a 10")
    # for i in range(start, end, increment)
    for i in range(0, fix_number+1, 1):
        print(i)
else:
    print("Stampo i valori da 0 a 10")
    for i in range(fix_number, -1, -1):
        print(i)

File: 2425/C++/primaEsercitazione.cpp
#include <iostream>
using namespace std;

int main()
{
    int val;

    cout << "Inserisci un valore intero: " << endl;
    cin >> val ;

    cout << "Il valore è: " << val << " " << endl;

    if(val > 10){
        cout << "Il valore è maggiore di 10" << endl;
        for(int i=0; i < 11; i++){
            cout << i << endl;
        }
    }
    else {
        cout << "Il valore è minore di 10" << endl;
        for(int i=10; i >= 0; i--){
            cout << i << endl;
        }
    }

    return 0;
}
